Classes/ViewHelpers/Format/TrimViewHelper.php

```diff
@@ -29,6 +29,6 @@ class TrimViewHelper extends AbstractViewHelper
      */
     public static function renderStatic(array $arguments, \Closure $renderChildrenClosure, RenderingContextInterface $renderingContext)
     {
-        return trim($renderChildrenClosure());
+        return trim($renderChildrenClosure()) ?? '';
```

Comment:
Sorry, I made an error when copy-pasting my patch around. It should be:
`return trim($renderChildrenClosure() ?? '');`
I don't know how to update a pull request.

Reply:
@fsuter 
Push to the branch you created the pull request from and it should be automatic updated.